source/delimiter-case.d.ts

```diff
@@ -87,13 +54,9 @@ const rawCliOptions: OddlyCasedProperties<SomeOptions> = {
 
 @category Change case
 @category Template literal
-*/
-export type DelimiterCase<Value, Delimiter extends string> = string extends Value ? Value : Value extends string
-	? StringArrayToDelimiterCase<
-	SplitIncludingDelimiters<Value, WordSeparators | UpperCaseCharacters>,
-	true,
-	WordSeparators,
-	UpperCaseCharacters,
-	Delimiter
-	>
+ */
+export type DelimiterCase<Value, Delimiter extends string, Options extends SplitWordsOptions = {splitOnNumber: true}> = Value extends string
```

Comment:
`SplitWordsOptions = {splitOnNumber: true}` will not allow us to add more options in the future, as if you specify one, you override the existing defaults.

Example: https://github.com/sindresorhus/type-fest/blob/7dfb307e3ecc0e7717de6880395f1de8715c82c1/source/paths.d.ts

Reply:
Added